assets/survey_themes/fruity/src/definitions.scss

```
@@ -2965,7 +2965,7 @@ body {
     }
 }
 
-/* Define the color for Awesome Bootsrap Checkboxes ; based on bg-primary background-color*/
+/* Define the color for Awesome Bootstrap Checkboxes ; based on bg-primary background-color*/
```

Stylelint (17.6.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)